app/views/Title.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import play.api.i18n.Messages
+
+object Title {
+
+  def createTitle(headingKey: String, sectionKey: Option[String] = None)(implicit messages: Messages): String =
```

Comment:
can we goes with default in `sectionKey` as empty string instead of Option - having empty would means not use it and leads to much easier call site

Reply:
Sure. I thought "" was the devils work?

Reply:
This could be a case class e.g 
```
case class Title(headingKey: String, sectionKey: Option[String] = None) {
    def toString(implicit messages: Messages) = ...
}
```

Then main template can be changed to only accept a `Title` type

Reply:
Adopted Ed's suggestion